### DISCUSSION TITLE
Adding an orgmode feature in nvchad as default setup

### DISCUSSION THREAD

**Discussion by Asrtodex** [NvChad/NvChad#1828] [Ideas]
many of the vim user love orgmode but it's hard to find a tutorial to add it on nvchad please add this feature as default because i love orgmode

**Comment by siduck** [comment]
@Asrtodex this is more of a user preference. I suggest you to just add the relevant plugins

**Reply by Asrtodex** [reply]
oc i just made to installed but it take some time to make it realy work because i dont know why the color scheme not work in org mode but now i have been able to make it working

**Reply by ustrucx** [reply]
Hi, do you mind please describing how you installed it and how you fixed the color? I am a newbie on nvim and having a though time with lazy plugin manager trying to install any plugin, tried neorg and it didn't work, definitely user error but the docs kinda assume one already know how to work with lazy and nvim, I really like NvChad but as a total newbie I am having trouble getting an org mode online. Thank you in advance!

**Reply by ustrucx** [reply]
.config/nvim/lua/custom/chadrc.lua

Add the `M.plugins = 'custom.plugins'` before `return M`:

` ---@type ChadrcConfig 
 local M = {}
 M.ui = {theme = 'nord'}
 M.plugins = 'custom.plugins'
 return M`

.config/nvim/lua/custom/plugins.lua

Add the following for Neorg:

`local plugins = {
  {
    "nvim-neorg/neorg",
    build = ":Neorg sync-parsers",
    dependencies = { "nvim-lua/plenary.nvim" },
    config = function()
      require("neorg").setup {
        load = {
          ["core.defaults"] = {}, -- Loads default behaviour
          ["core.concealer"] = {}, -- Adds pretty icons to your documents
          ["core.dirman"] = { -- Manages Neorg workspaces
            config = {
              workspaces = {
                home = "~/Documents/personal/notes",
                work = "~/Documents/professional/chainalysis/notes",
              },
            },
          },
        },
      }
    end,
  },
}
return plugins`

Restart nvim, run the following command:

`:Lazy`

Then Shift+I
